style.css

```diff
@@ -0,0 +1,435 @@
+.container {
+  background: url(./img/background.jpg);
+  background-size: cover;
+  display: flex;
+  max-width: 1440px;
+  min-width: 320px;
+  height: 100vh;
+  margin: auto;
+}
+
+.menu { margin: 0 25px 0 100px; }
+
+.title {
+  color: #fff;
+	margin: 50px 0 20px 0;
+	font: normal bold 57px/ 67px 'Roboto', sans-serif;
+}
+
+.subtitle {
+  color: #fff;
+  margin: 0 0 60px 0;
+  font: normal bold 19px/ 21px 'Roboto', sans-serif;
+}
+
+.menu__level-selection {
+  position: relative;
+  margin-bottom: 70px;
+}
+
+.level_btn {
+  display: flex;
+  margin-bottom: 20px;
+  position: relative;
+  background: none;
+  border: none;
+  color: #fff;
+	font: normal bold 50px/ 59px 'Roboto', sans-serif;
+  outline: none;
+  padding-left: 40px;
+}
+
+.level_btn:hover {
+  opacity: .8;
+  cursor: pointer;
+  position: relative;
+}
+
+.level_checked::before {
+  content: '';
+  background-color: #d62b2e;
+  position: absolute;
+  top: 20px;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  transform: rotate(45deg);
+}
+
+.begin-button {
+  background: #d62b2e;
+  border: none;
+  border-radius: 7px;
+  color: #fff;
+  font: normal bold 32px/ 37px 'Roboto', sans-serif;
+  height: 90px;
+  width: 350px;
+  outline: none;
+}
+
+.begin-button:hover {
+  cursor: pointer;
+  opacity: .9;
+}
+
+.images {
+  position: absolute;
+  top: 40%;
+  right: 8%;
+}
+
+.first-card {
+  background-image: url(./img/card.png);
+  height: 341px;
+  width: 218px;
+  transform: rotate(11.9deg);
+}
+
+.second-card {
+  background-image: url(./img/card.png);
+  height: 341px;
+  width: 218px;
+  position: relative;
+  right: 45px;
+  top: 20px;
+  transform: rotate(39.26deg);
+}
+
+/* style js */
+
+.card-wrap {
+  cursor: pointer;
+  height: 340px;
+  width: 220px;
+  position: relative;
+  transform-style: preserve-3d;
+	transition: transform .8s;
+}
+
+.card-wrap__hover:hover {
+	box-shadow: 10px 10px 15px rgba(0, 0, 0, 0.5);
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/finaljs/pullRequest?prid=5949810)